editors/code/src/installation/interfaces.ts

```diff
@@ -42,6 +41,9 @@ export namespace ArtifactSource {
          */
         repo: GithubRepo;
 
+
+        // FIXME: add installationPath: string;
```

Comment:
This will simplify code a bit, will do it in separate PR in order not to bloat this one.